includes/utilities.inc

```diff
@@ -130,7 +130,10 @@ function islandora_entities_get_related($identifier, $title, $type) {
       'islandora_entities_thesis_base_query',
       '+RELS_EXT_hasModel_uri_mt:"ir:thesisCModel"'),
```

Comment:
This is outside the range of this PR, but in the same vein of de-hard-coding solr fields, and not a required change for this PR: RELS_EXT_hasModel_uri_mt could (should?) be replaced by the variable islandora_solr_content_model_field, and the pids of the content models ("ir:thesisCModel") replaced by their URIs ("info:fedora/ir:thesisCModel"). This is already used in this module, in line 37 of utilities.inc, though see also the requisite nasty string formatting from lines 58-63.